libs/react/src/lib/segmented-control/segmented-control.tsx

```diff
@@ -36,10 +36,10 @@ export interface SegmentProps {
   disabled?: boolean
   children?: ReactNode | ReactNode[]
   width?: string
-  'min-Width'?: string
-  'max-Width'?: string
+  minWidth?: string
+  maxWidth?: string
   /** @deprecated */
-  segMinWidth?: number
+  segMinWidth?: string
 }
```

Comment:
I'm guessing this change is because you need to specify `px` for this to work now, but this is a breaking change to an already deprecated property. The point of deprecating it in the first place was to avoid breaking changes, so t would be better to just add `px` to it in the fallback on line 69.

The changes to minWidth/maxWidth are technically breaking too, but since they were just added and the component itself is deprecated, that change is unlikely to cause trouble, so I think we can keep that.

Reply:
@splashdust fixed.